### DISCUSSION TITLE
pass a number to the res.writeStatus

### DISCUSSION THREAD

**Discussion by muturgan** [uNetworking/uWebSockets.js#193] [Q&A] [answered]
this method waiting for a string argument
it allows you to pass a "some" for example instead of "200"
in this case i see a 200 status in a browser
and an "unsupported protocol" error in the Insomnia rest client

i suggest to change the expected argument data type to a number

**Comment by ghost** [comment]
It takes a string but you could declare a bunch of predefined values to pick from

**Comment by dalisoft** [comment]
@muturgan You can see how to implement this from [here](https://github.com/dalisoft/nanoexpress/blob/master/src/proto/http/polyfill-chunks/status.js#L11-L12)

**Comment by ghost** [comment, marked as answer]
It takes a string because you are supposed to pass "200 OK" or something like that. You can PR an enum with a few common statuses if you want to

**Reply by santinalbrowns** [reply]
Can you give an example for a 400 status code!

I tried to do it like res.writeStatus("400 Bad Request"); but am still getting a 200

**Reply by uNetworkingAB** [reply]
You're most likely writing status in the wrong order. Check examples, you must write status first of all
